gtwrap/pybind_wrapper.py

```diff
@@ -277,6 +295,19 @@ def wrap_stl_class(self, stl_class):
                     stl_class.properties, cpp_class),
             ))
 
+    def wrap_enum(self, enum, prefix='\n' + ' ' * 8):
+        """Wrap an enum."""
+        namespaces = enum.namespaces()
+        # leverage Typename to get the fully-qualified name easily.
+        name = parser.Typename(namespaces + [enum.name]).to_cpp()
+        res = "    py::enum_<{name}>(m_, \"{enum.name}\", py::arithmetic())".format(
+            enum=enum, name=name)
+        for enumerator in enum.enumerators:
+            res += "{prefix}.value(\"{enumerator.name}\", {name}::{enumerator.name})".format(
```

Comment:
nit, but why not use single-quotes on the outside to avoid having to escape the double-quotes on the inside?

Reply:
I like the guarantees that double quotes gives. I don't know if single quote behavior has been updated since Python 3.5 but I like having consistency.

Reply:
not sure I understand this - in python I think single quotes and double quotes are and "always" have been (at least since 2.4) interchangeable?  Not sure which guarantee you're referring to?

Reply:
Back in the day there was something about not being able to use formatting with single quotes. Maybe that's changed, but I like double quotes for strings and single quotes for characters since it is consistent across languages.

Reply:
hmm ok, sounds good.

Just for your future reference, [python 1.4 documentation](https://docs.python.org/release/1.4/ref/ref2.html#HDR14) states that string literals can use `'` or `"` (interchangeably) so anywhere a there is a string literal (including f-strings, which are string literals prefixed by `f`) you can feel free to use either version.

Also, this is maybe too aggressive of a comment, but checkout [PEP 8 formatting guidelines](https://www.python.org/dev/peps/pep-0008/#string-quotes).  [google style guide](https://google.github.io/styleguide/pyguide.html#310-strings) allows but not require switching `'` vs `"` to avoid escaping.

Reply:
Those guidelines are what I already do 😂 I use " exclusively but switched to ' in this PR to remove the backslash.